### PR TITLE
ci: associate `install.sh` with used job

### PR DESCRIPTION
**Pull request by Borda**
This is just a minor cleaning to reduce root structure.
The `install.sh` is used in a single place in GH actions. I would suggest moving it to the same folder.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -3,12 +3,8 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
   cancel-in-progress: true
 on:  # yamllint disable-line rule:truthy
-  push:
-    branches:
-      - '*'
-  pull_request:
-    branches:
-      - '*'
+  push: {}
```
Comment (Borda) [inline .github/workflows/tests.yml:6]: Note the meaning is the same; on the other hand, I would suggest running them only against the master to reduce the number of jobs running in parallel the PR comes from the root repo

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -3,12 +3,8 @@ concurrency:
...
+  pull_request: {}
```
Comment (Borda) [inline .github/workflows/tests.yml:7]: Note the meaning is the same; on the other hand, I would suggest running them only against the master to reduce the number of jobs running in parallel the PR comes from the root repo

```suggestion
 push:
    branches: ['master']
  pull_request:
    branches: ['master']
```

**Review by lucyleeow — APPROVED** [review]
Makes sense now that we are only using GH actions. I'll let @larsoner merge as he's more familiar with the CIs. 

Thanks @Borda !

**Review by larsoner — APPROVED** [review]
Thanks @Borda

**Pull request merged**